src/ucs/sys/sock.c

```diff
@@ -101,14 +101,31 @@ ucs_status_t ucs_socket_setopt(int fd, int level, int optname,
     return UCS_OK;
 }
 
+static const char *ucs_socket_getname_str(int fd, char *str, size_t max_size)
+{
+    struct sockaddr_storage sock_addr = {0};
+    socklen_t addr_size;
+    int ret;
+
+    addr_size = sizeof(sock_addr);
+    ret       = getsockname(fd, (struct sockaddr*)&sock_addr,
+                            &addr_size);
+    if (ret < 0) {
+        ucs_debug("getsockname(fd=%d) failed: %m", fd);
+        ucs_strncpy_zero(str, "-", max_size);
```

Comment:
ucs_strncpy_zero is overkill because it fills up to max_size. can use ucs_strncpy_safe instead which only takes care of 0 terminator